datahub-web-react/src/app/permissions/policy/ManagePolicies.tsx

```diff
@@ -166,14 +166,14 @@ export const ManagePolicies = () => {
         data: policiesData,
         refetch: policiesRefetch,
     } = useListPoliciesQuery({
-        fetchPolicy: 'no-cache',
         variables: {
             input: {
-                start,
+                start: query && start === 0 ? null : start,
                 count: pageSize,
-                query,
+                query: (query?.length && query) || undefined,
```

Comment:
can you remind me why we are having to change this? i'd expect to only see reset in the start variable any time that the query changes

Reply:
my bad it work same i will update

Reply:
remove unwanted code